tools/azure-sdk-tools/packaging_tools/generate_utils.py

```diff
@@ -435,7 +435,6 @@ def generate_ci(template_path: Path, folder_path: Path, package_name: str) -> No
 
 def gen_typespec(typespec_relative_path: str, spec_folder: str, head_sha: str, rest_repo_url: str) -> Dict[str, Any]:
     typespec_python = "@azure-tools/typespec-python"
-    autorest_python = "@autorest/python"
```

Comment:
After 0.25.0, typespec-python doesn't rely on `autorest.python` so no need to record the version